src/Microsoft.Health.Fhir.CosmosDb/Features/Storage/Operations/CosmosFhirOperationDataStore.cs

```diff
@@ -347,9 +347,35 @@ public async Task<bool> CheckActiveReindexJobsAsync(CancellationToken cancellati
             }
         }
 
-        public Task<ReindexJobWrapper> GetReindexJobByIdAsync(string jobId, CancellationToken cancellationToken)
+        public async Task<ReindexJobWrapper> GetReindexJobByIdAsync(string jobId, CancellationToken cancellationToken)
```

Comment:
Nit: On line 294 there is a typo, an/a. For some reason GitHub won't let me put a comment there,